### ISSUE TITLE
load assets per template

### ISSUE THREAD

**Issue by selul** [Codeinwp/wp-product-review#209]
right now almost the frontend templates are fully separated from the backend logic, and those can be replaced in the wppr folder, inside the theme/child theme. 

The only two views that are not using the templater class are those for the comment form and comment rating ( the view which displays the user's rating after he rated the product ) 

We will need to also use the templer class for those two. 

After this is done, we can leverage the use of this class and load assets based on the templates used. 

For instance, in the templater class, i would like before rendering the template, to trigger the enqueue of the  assets needed for that specific template. 

Basically all the public facing assets will be loaded thru the templater class also. 

For instance when we load the default template, we will trigger the loading of the assets needed for that specific template.

**Comment by contactashish13** [comment]
@selul some assets are generic and not dependent on the template e.g. lightbox assets. do you want these to be also loaded from the template class or can they remain where they are?

**Comment by selul** [comment]
lightbox is dependent by default template. 

for all that are used across all templates, yes, i would like to load with the template class.

**Comment by contactashish13** [comment]
@selul https://github.com/Codeinwp/wp-product-review/pull/213
